### ISSUE TITLE
No logfiles visible on web-gui

### ISSUE THREAD

**Issue by seiha011** [mbentley/docker-ums#23]
Thank you for providing the ums-docker-image for arm64. I started it as suggested, but I can't see any log files in the web interface. In the environment I set the log level to trace. No change. Do you have a tip for me?

**Comment by mbentley** [comment]
In this case, it believe it would be because the logs are only being streamed to stdout/stderr of the container and are not being written to disk which I imagine the web UI is trying to pull them from.  You would need to view the logs through Docker's interfaces.

**Comment by seiha011** [comment]
OK I will find them... /var/log ....  ;-)  I think normally the debug.log is written to /home/user/.config/UMS/debug.log.   No problem. Thanks!

**Comment by mbentley** [comment]
I haven't ran UMS for a while but still keep this repo maintained so my recollection of the behavior is a bit iffy so I did a quick test and it looks like some debug logs are being written to: `/var/log/UMS/ums/debug.log` which is in a [volume by default](https://github.com/mbentley/docker-ums/blob/master/Dockerfile.14-alpine#L57).  If you didn't map that to a path, it would just be a docker managed volume.  The typical INFO logs are just in what can be retrieved with `docker logs`.

Not sure if there is anything I can do to change the behavior of the app in the container that would allow the logs to stream through the UI or not - not sure how that works.

**Comment by seiha011** [comment]
Yes thanks .. You are right, it's an ums-thing. i had some issues with an old renderer, so i tried to get the log-files via the web-interface. I will get it now via cli and docker ... or i will find another solution  ;-)    regards
